### PR TITLE
[11.0][FIX] web_action_conditionable: Adapt to new api

### PR DESCRIPTION
**Pull request by Tardo**
Current implementation of "web_action_conditionable" doesn't work because 'web.View' not exists.
This PR refactor all to use new api.

Affected Versions: 11.0 & 12.0

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @Yajo I think you could please review this PR, thanks!

File: web_action_conditionable/static/src/js/field_one2many.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* global py */
+/* Copyright 2019 Alexandre Díaz
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
```
Comment (nikul-serpentcs) [inline web_action_conditionable/static/src/js/field_one2many.js]: @Tardo LGPL -> AGPL
As Per README file

Comment (Tardo) [reply web_action_conditionable/static/src/js/field_one2many.js]: Change done, thanks.

File: web_action_conditionable/static/src/js/field_one2many.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "use strict";
+
+    var FieldOne2Many = require('web.relational_fields').FieldOne2Many;
+
+    FieldOne2Many.include({
+        init: function () {
+            var self = this;
+            try {
+                return this._super.apply(this, arguments);
+            } catch (error) {
```
Comment (yajo) [inline web_action_conditionable/static/src/js/field_one2many.js:14]: I don't understand this structure. Are you only adding your stuff when upstream code fails? Why not adapt those attributes to booleans and avoid `_super()` to fail? :thinking:

**Comment by pedrobaeza** [comment]
Please rebase and answer the question, but if it gets complicated, we don't need this at all, so we should close it.

File: web_action_conditionable/static/src/js/field_one2many.js (hunk 2 above)
Comment (Tardo) [reply web_action_conditionable/static/src/js/field_one2many.js:14]: This follows the original implementation. This allows to have the "normal" behavior and the "plus" added by this module.
I don't know how can be converted to boolean before initialize fields...

**Comment by Tardo** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
OK, merging then /ocabot merge minor

**Pull request merged**